saleor/account/forms.py

```diff
@@ -40,66 +39,66 @@ def get_address_form(
         address_form = address_form_class(data, instance=instance, **kwargs)
     else:
         initial_address = (
-            initial if not preview
-            else data.dict() if data is not None else data)
+            initial if not preview else data.dict() if data is not None else data
```

Comment:
Black supports pass max line length by ```--line-length``` as mentioned in https://github.com/ambv/black#line-length. Saleor is using flake8's default 79, should we pass this argument as 79 as well?

Reply:
I'm also fine with using Black's default settings here.